assets/sass/modules/_enroll.scss

```diff
@@ -430,10 +430,6 @@ $enroll-content-spacing: 15px;
       }
     }
   }
-  .chzn-drop:focus-within {
-    box-shadow: 0 0 8px $input-shadow-color-focus;
-    border-color: $input-border-color-focus;
-  }
```

Comment:
is this new change? I dont see it in the respective [okta-ui PR](https://github.com/okta/okta-ui/pull/4940) will removing this cause any regression?

Reply:
this change is specific to SIW. none of the chosen select component has focus highlighted in SIW. especially when working with screen 
readers. Without focus, it's hard to figure out where the cursor is. 


<img width="810" alt="Screen Shot 2022-03-02 at 2 04 33 PM" src="https://user-images.githubusercontent.com/94395085/156430697-42cc3135-3e86-4ce0-b85d-e70243d4f153.png">

Reply:
btw its the same highlight styles we have for input text box
<img width="456" alt="Screen Shot 2022-03-02 at 2 05 46 PM" src="https://user-images.githubusercontent.com/94395085/156430851-4b69c4da-a21b-4a9c-b689-f9c8c0526214.png">
.

Reply:
is `.chzn-drop` no longer exist on the UI and so`.chzn-drop:focus-within` is being removed?

Reply:
`.chzn-drop` is for the dropdown options and will have height and width only when select is open. we need to have focus even when the select is closed, so i have moved the rules to it's parent `.chzn-container`